### PR TITLE
chore: Update API endpoint URL

### PR DESCRIPTION
**Pull request by power-f-GOD**
Resolves: https://github.com/clearpool-finance/clearpool.finance/issues/1294
Resolves: https://github.com/clearpool-finance/clearpool.finance/issues/1435

Parent: https://github.com/clearpool-finance/DefiLlama-Adapters/pull/3

TA: `3hrs`

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/clearpool exports TVL: 
        
 
 
    ethereum-staking          5.42 M
    ethereum                  4.76 M
    ethereum-borrowed         19.12 M
    polygon                   79.02 k
    polygon-borrowed          0.016955049308
    staking                   5.42 M
    borrowed                  19.12 M
    
    total                    4.84 M

**Comment by georgeciubotaru** [comment]
@power-f-GOD Please sync also,  our repo with the `upstream` repo

**Comment by georgeciubotaru** [comment]
> @power-f-GOD Please sync also, our repo with the `upstream` repo

@power-f-GOD My bad, haven't seen the commit.

Great work! 👍

File: projects/clearpool/index.js [hunk 1]
```diff
@@ -80,36 +78,40 @@ const polygonTvl = async (timestamp, _,  { polygon: block }) => {
...
+    )
+  );
   return totalBorrowed;
 };
 
-
-const singleStakingContracts = [
-  "0x629E39da1Db5654fe59cAE31d48CAEBB8dC2A9c6",
-];
+const singleStakingContracts = ["0x629E39da1Db5654fe59cAE31d48CAEBB8dC2A9c6"];
```
Comment (georgeciubotaru) [inline projects/clearpool/index.js:114]: @power-f-GOD Let's define also the `zkevm` network the properties

**Comment by power-f-GOD** [comment]
@georgeciubotaru , `zkEVM` doesn't have a `staking` network in the Protocol only `membershipStaking`:

```ts
    ...
    1101: {
      eth: '0x0000000000000000000000000000000000000000',
      poolFactory: '0xCE3Fec90A05992dF1357651FEF6D143FeeC7Ca16',
      auction: '0x5cda11bb36743fA3F2215828e744967854255900',
      membershipStaking: '0x3B0bbd7E5877d64aD5886dDe14ca5cEecC29D55B',
      lens: '0xA919D35747fa83f88670B79fe5804127aD9584E3',
      cpool: '0xc3630b805F10E91c2de084Ac26C66bCD91F3D3fE',
      usdc: '0xA8CE8aee21bC2A48a5EF670afCc9274C7bbbC035',
      usdt: '0x1E4a5963aBFD975d8c9021ce480b42188849D41d',
      weth: '0x4F9A0e7FD2Bf6067db6994CF12E4495Df938E6e9',
      wbtc: '0xEA034fb02eB1808C2cc3adbC15f447B93CbE08e1',
      multicall: '0xcA11bde05977b3631167028862bE2a173976CA11'
    },
    ...
```

**Comment by power-f-GOD** [comment]
DefiLlama Adapters PR: https://github.com/DefiLlama/DefiLlama-Adapters/pull/7005

**Comment by georgeciubotaru** [comment]
@power-f-GOD `staking` is just for `ETH` mainnet

**Comment by power-f-GOD** [comment]
> @power-f-GOD `staking` is just for `ETH` mainnet

Ok. What am I to do on that line of code?

**Comment by georgeciubotaru** [comment]
@power-f-GOD Just implement `tvl` and `borrowed` amounts

**Comment by power-f-GOD** [comment]
Ok. Been trying to understand the code... https://github.com/clearpool-finance/DefiLlama-Adapters/blob/31f70165bbf1bd83a8f9b5854020cef6ba75b043/projects/clearpool/index.js

`ethereumTVL` doesn't use the pools endpoint URL, but `polygonTVL` does. So, what would it be for `zkEVMTVL`?

Edit: Why is it that way if I may ask?

**Pull request merged**